pepper-apis/src/main/java/org/broadinstitute/ddp/model/event/ActivityInstanceCreationEventAction.java

```diff
@@ -101,7 +102,7 @@ public void doAction(Handle handle, EventSignal signal) {
         }
 
         Long parentInstanceId = null;
-        if (signal.getEventTriggerType() == EventTriggerType.ACTIVITY_STATUS) {
+        if (activityDto.getParentActivityId() != null && signal.getEventTriggerType() == EventTriggerType.ACTIVITY_STATUS) {
```

Comment:
I was missing the check for parent activity. Without this check, we're always setting the `parentInstanceId`, which means any instances we create from this event will be effectively a "child" instance and won't show up on the dashboard.